release-notes/swan-lake-2201.1.0-release-note.md

```diff
@@ -30,9 +30,368 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### New Features
 
+##### The `spread` operator support for `list` constructor
+
+Introduced the `spread` operator support for the `list` constructor expression.
```

Comment:
```suggestion
Introduced the spread operator support for the list constructor expression.
```

Reply:
Emphasis needs to be fixed in line with https://github.com/ballerina-platform/ballerina-release/pull/1961#discussion_r870393444.

Reply:
We don't need the "the" before spread either.

Reply:
@gimantha we need to commit the above as well.